### ISSUE TITLE
No visual indication when keyboard navigation disabled

### ISSUE THREAD

**Issue by cpcallen** [rachel-fenichel/blockly-keyboard-experimentation#92]
At the moment, when you press 'enter' to enable keyboard navigation the cursor appears, but there is no corresponding visual indication when keyboard navigation is disabled (i.e. you press 'esc').

There should be a clear visible change when keyboard navigation is disabled; at minimum, the cursor should be hidden.

Currently we are using block selection as the main visual indication the cursor is on a block.  If we continue to do that we should have an additional visual indicator (e.g. a small arrow) to denote focus; this additional indicator can be hidden even if the block remains selected (and would additionally distinguish the focused block in the event that multiselect is supported).

In the event that we continue to oblige use of enter/esc to enable and disable keyboard navigation, pressing esc should not result in the workspace focus indicator (blue / black outline) disappearing—in fact, perhaps it should _reappear_ when esc is pressed, having previous disappeared (in favour of the cursor) when enter was pressed.
